### PR TITLE
People list page

### PR DESCRIPTION
**Pull request by Vov4ukz53**
I added people list page! There are a lot of changes and commits, because i wanted to make these pages look good. 
Please check and express your opinion.

### CODE REVIEW

File: src/common/Header/styled.js [hunk 1]
```diff
@@ -85,23 +83,31 @@ export const ListItem = styled.li`
...
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    padding: 6px 12px;
+    margin: 0 2px;
+  }
 
   @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
     font-size: 12px;
   }
+
+  &.active{
```
Comment (Gosia-Ras) [inline src/common/Header/styled.js:107]: 💯

File: src/features/movies/MovieList/index.js [hunk 2]
```diff
@@ -21,19 +26,29 @@ export const MovieList = () => {
...
-            poster={`${apiUrlImage}w500/${movie.poster_path}`} //here we need to change the image for mobile phones 
-            title={movie.title}
-            year={(movie.release_date).slice(0, 4)}
-            genres={movie.genre_ids}
-            rating={movie.vote_average}
-            voteCount={movie.vote_count}
-          />
-        })}
-      </Wrapper>
+      {loading
```
Comment (Gosia-Ras) [inline src/features/movies/MovieList/index.js:29]: Nice!

File: src/features/people/peopleList/PersonTile/styled.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
```
Comment (Gosia-Ras) [inline src/features/people/peopleList/PersonTile/styled.js:3]: This is very similar to Wrapper from MovieTile, have you tried to add another attribute that would just apply the styles that are different?

File: src/features/people/peopleList/PersonTile/styled.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    width: 100%;
+    flex-direction: column;
+    height: 100%;
+
+    @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+        padding: 8px;
+    }
+`;
+
+export const Poster = styled.img`
```
Comment (Gosia-Ras) [inline src/features/people/peopleList/PersonTile/styled.js:18]: same as above, Poster from MovieTile is similar

**Review by Gosia-Ras — COMMENTED** [review]
You did a great job! 🚀 
I added some small comments.

File: src/features/people/peopleList/peopleListSlice.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    fetchPeople,
+    setPeople,
+    setError,
+    setPage,
+} = peopleListSlice.actions;
+
+const selectPeopleListState = state => state.people;
+
+export const selectPeopleList = state => selectPeopleListState(state).people;
+export const selectPeoplePage = state => selectPeopleListState(state).pagge;
```
Comment (olachrzan) [inline src/features/people/peopleList/peopleListSlice.js]: You have a typo here 🙃 (pagge instead of page)

File: src/common/wrapperLink.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+export const WrapperLink = styled(Link)`
+    text-decoration: none;
+    color: ${({ theme }) => theme.colors.darkText};
+    transition: transform .4s;
+    
+    &:hover{
+        transform: scale(1.04);
```
Comment (olachrzan) [inline src/common/wrapperLink.js:10]: It looks really good 😃

File: src/features/people/peopleList/peopleListSlice.js (hunk 5 above)
Comment (Vov4ukz53) [reply src/features/people/peopleList/peopleListSlice.js]: I will fix it, thank you

File: src/features/people/peopleList/peopleListPage.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  const people = useSelector(selectPeopleList);
+  const loading = useSelector(selectPeopleLoading);
+  console.log(loading);
+
+  useEffect(() => {
+    dispatch(fetchPeople());
+  }, [dispatch])
+
+  return (
+    < Container >
```
Comment (olachrzan) [inline src/features/people/peopleList/peopleListPage.js:25]: All that's left to do is add a Title from common

**Comment by olachrzan** [comment]
You worked really hard. It looks great 🎉

File: src/features/people/peopleList/PersonTile/styled.js (hunk 3 above)
Comment (Vov4ukz53) [reply src/features/people/peopleList/PersonTile/styled.js:3]: we will discuss this

File: src/features/people/peopleList/PersonTile/styled.js (hunk 4 above)
Comment (Vov4ukz53) [reply src/features/people/peopleList/PersonTile/styled.js:18]: we will discuss this

**Pull request merged**